includes/bootstrap/SWSDrupalContext.php

```diff
@@ -70,24 +70,28 @@ public function login() {
       throw new \Exception('Tried to login without a user.');
     }
 
-    $this->getSession()->visit($this->locatePath('/user'));
-    $element = $this->getSession()->getPage();
-    // find the Local User Login link - it's only findable in the browser, with Javascript
-    // See Behat\Mink\Element\TraversableElement::findLink
-    $localuserlogin = $element->findLink("Local User Login");
-    // If we can find a link, that means we're using Javascript.
-    if (!is_null($localuserlogin)) {
+
+    // @todo, find out if there is a more concrete way of if JS is enabled.
+    $session = $this->getSession();
+    $element = $session->getPage();
+    $javascript_enabled = method_exists($session, 'executeScript');
```

Comment:
The other option would be to check for the selenium instance of the driver but it is possible they we swap selenium for some other sort of JS enabled driver in the future.